jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -126,6 +128,15 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun addPayloadType(payloadType: PayloadType) {
+        transceiver.addPayloadType(payloadType)
+        bitrateController.addPayloadType(payloadType)
+    }
+
+    override fun addRtpExtension(rtpExtension: RtpExtension) {
+        transceiver.addRtpExtension(rtpExtension)
```

Comment:
You didn't want to use `=` notation for this?

Reply:
Sure, fixed.